src/hooks/useContinueWatchingList.tsx

```diff
@@ -1,29 +1,82 @@
 import { useState, useCallback, useRef } from 'react';
 import { useFocusEffect } from '@react-navigation/core';
-import { getListOfUniqueEventId } from '@services/bitMovinPlayer';
+import { getListOfWatchedVideos } from '@services/bitMovinPlayer';
+import { useAppSelector } from './redux';
+import { customerIdSelector } from 'services/store/auth/Selectors';
 
 export const useContinueWatchingList = (): {
   data: Array<string>;
   ejected: boolean;
 } => {
-  const ejected = useRef<boolean>(false);
+  const customerId = useAppSelector(customerIdSelector);
+  const { eventsLoaded } = useAppSelector(state => state.events);
+
+  const createVideoToEventMap = useAppSelector(
```

Comment:
This Video-Event map is a bit of a hack, but I couldn't think of a better way to implement this. The store doesn't have a precise 1:1 mapping from Diese ID to event ID, so I had to work with this.

I also figured that modifying the store wouldn't have been feasible, since there are many things tightly coupled and the risk of breaking something would've been pretty big.

Let me know if you think there's a better way, but honestly this works pretty well as it is.

Not particularly elegant, but gets the job done.